packages/ra-ui-materialui/src/input/SelectInput.js

```diff
@@ -198,14 +201,15 @@ export class SelectInput extends Component {
                     />
                 }
                 className={`${classes.input} ${className}`}
+                clearAlwaysVisible
```

Comment:
why?

Reply:
Used by SelectInput, because showing the clear button only when focused is unusable as focus open the menu